index.js

```diff
@@ -115,23 +117,81 @@ app.post('/customers/create', async (request, result) => {
   }
 })
 
-// Endpoint used to create a new inventory item.
-// Contains a site id (the site already exists in airtable).
+// Endpoint to create a new inventory item and an initial inventory update.
+// Assumes that the site, user, and product already exist in Airtable.
```

Comment:
NIT: can you comment that the custom endpoint is needed to account for offline functionality (must be able to create both items in a single endpoint)?